. Added Authors Details screen

alsaidmandor-eng opened the pull request:
. Added pram in route to authors details
. Added modify in author list view to use in horizontal or vertical and pass list for every writer
![authors_details_screen](https://github.com/user-attachments/assets/bc27935a-772f-4f3a-9785-1c1cbbc1ca49)✅✅👍

alsaidmandor reviewed (approved):
Good performance 👍

alsaidmandor commented:
Ok

On Mon, Aug 26, 2024, 3:00 PM alsaidmandor-eng ***@***.***>
wrote:

> @alsaidmandor-eng <https://github.com/alsaidmandor-eng> requested your
> review on: #22 <https://github.com/alsaidmandor/bazar/pull/22> . Added
> Authors Details screen.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alsaidmandor/bazar/pull/22#event-14012429270>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMRA7MKMG5VCSN5OGSPNBZTZTMKE3AVCNFSM6AAAAABND3TOKWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJUGAYTENBSHEZDOMA>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>

alsaidmandor committed:
. Added Authors Details screen
. Added pram in route to authors details
. Added modify in author list view to use in horizontal or vertical and pass list for every writer
